build.gradle.kts

```diff
@@ -71,6 +71,12 @@ tasks.jar.configure {
   }
 }
 
+tasks.withType<JavaCompile>().configureEach { 
```

Comment:
Is it correct to understand the purpose of this change as better visibility of deprecation warnings?

Reply:
Yes, without this you'll just get "there was deprecated API usage", but it doesn't actually tell you which file or where. If it's always shown, it's more likely to get fixed.

After you merge all my PRs the only warning that stays will be the Maven publishing usage, which will be migrated soon too (there's a PR and I'm willing to revive that later). After that, if you're willing, you can treat warnings as errors, so no new deprecations can be introduced. If something gets deprecated (which will happen during a library upgrade), it can be fixed immediately, because it would break the build. This brings warning fixes to the discovery point, rather than wait for them to be errors, because the function was removed.

Reply:
Gotcha, thanks for the explanation! I just noticed this change in #448 as well - let’s remove from this PR and make it there then.

Reply:
https://github.com/GoogleCloudPlatform/app-gradle-plugin/pull/451/commits/03ca43169568d17ca3032325ac6f14edad1da31f